### PR TITLE
Redirect dist_pipeline and pipeline_tutorials tutorials to pipelining_tutorial

### PR DESCRIPTION
**Pull request by svekars**

### CODE REVIEW

File: intermediate_source/pipelining_tutorial.rst [hunk 1]
```diff
@@ -1,236 +1,11 @@
 Introduction to Distributed Pipeline Parallelism
```
Comment (wz337) [inline intermediate_source/pipelining_tutorial.rst:1]: This is up-to-date. We can keep this.

File: intermediate_source/dist_pipeline_parallel_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Distributed Pipeline Parallelism Using RPC
+==========================================
+
+This tutorial has been deprecated.
+
+Redirecting in 3 seconds...
+
+.. raw:: html
+
+   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/tutorials/index.html'" />
```
Comment (wz337) [inline intermediate_source/dist_pipeline_parallel_tutorial.rst:10]: We can redirect users to the new pipeline tutorial. https://pytorch.org/tutorials/intermediate/pipelining_tutorial.html

**Review by wz337 — APPROVED** [review]
LGTM!

File: intermediate_source/pipeline_tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Training Transformer models using Pipeline Parallelism
+======================================================
+
+This tutorial has been deprecated.
+
+Redirecting to the latest parallelism APIs in 3 seconds...
```
Comment (wz337) [inline intermediate_source/pipeline_tutorial.rst:6]: Let's also redirect to the pipeline tutorial above.

File: intermediate_source/pipeline_tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+.. raw:: html
+
+   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/tutorials/beginner/dist_overview.html#parallelism-apis'" />
```
Comment (svekars) [inline intermediate_source/pipeline_tutorial.rst:10]:
```suggestion
   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/tutorials/intermediate/pipelining_tutorial.html'" />
```

**Pull request merged**